client/state/data-layer/wpcom/me/two-step/application-passwords/index.js

```diff
@@ -48,6 +56,7 @@ const requestHandler = {
 			fetch: requestApplicationPasswords,
 			onSuccess: handleRequestSuccess,
 			onError: noop,
+			fromApi,
```

Comment:
Just noting that we have [`makeParser`](https://github.com/Automattic/wp-calypso/blob/3e1cd82f9895d9ca03bfd98d5e03b1220ab6e076/client/state/data-layer/wpcom-http/utils.js#L69-L97).

I think validating all the data we receive before it gets into state is a good thing and should be encouraged. With `makeParser` we could add a schema and ensure that `appPasswords` has the shape and types we expect before the arrive in our redux tree.

If you think that's overkill, feel free to go ahead with this.

Reply:
Sounds like a good idea to add some more strict validation by using a schema. 

Added a task to #22912 for this, so I'll handle it in a separate PR.